### PR TITLE
Ft article comments 159961957

### PR DESCRIPTION
**Pull request by Tittoh**
#### What does this PR do?
Adds comments feature

#### Description of Task to be completed?
Create and fetch comments

#### How should this be manually tested?
```
git fetch origin ft-article-comments-159961957

git checkout ft-article-comments-159961957

yarn install

yarn start

yarn test
```
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?

[#159961957 ](https://www.pivotaltracker.com/story/show/159961957) Users should be able to comment on articles

#### Screenshots (if appropriate)
screen shot 2018-09-20 at 10 03 40

### CODE REVIEW

File: src/components/CommentHead/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import capitalize from '../../utils/capitalize';
+
+const CommentHead = ({ user, date }) => (
+  <div className="row comment-box__user">
+    <img src={user.image} alt="" className="responsive-img circle" />
```
Comment (yeenab) [inline src/components/CommentHead/index.js]: Also if there is no image why don't we show an avatar here too

File: src/components/CommentHead/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      </p>
+    </div>
+  </div>
+);
+
+export default CommentHead;
+
+CommentHead.propTypes = {
+  user: PropTypes.shape({
+    username: PropTypes.string.isRequired,
```
Comment (yeenab) [inline src/components/CommentHead/index.js:31]: Validate a user for image here too

File: src/components/CommentInput/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+import capitalize from '../../utils/capitalize';
+import config from '../../config';
+
+const CommentInput = ({
+  type, name, label, onChange, handleSubmitComment, value, errors, failure, user,
+}) => (
+  <div className="comment-box">
+    <div className="row comment-box__user">
+      <img src={user.image ? user.image : config.DEFAULT_USER_AVATAR} alt="" className="responsive-img circle" />
```
Comment (yeenab) [inline src/components/CommentInput/index.js:12]: I believe we will get an error here if the user is not present. I just got it in a few and it disappeared

**Review by yeenab — CHANGES_REQUESTED** [review]
### NitPick

File: src/components/CommentHead/index.js (hunk 1 above)
Comment (Tittoh) [reply src/components/CommentHead/index.js]: I have added one

File: src/components/CommentHead/index.js (hunk 2 above)
Comment (Tittoh) [reply src/components/CommentHead/index.js:31]: Worked on it

File: src/components/CommentInput/index.js (hunk 3 above)
Comment (Tittoh) [reply src/components/CommentInput/index.js:12]: It happens on the localhost, so you have to refresh to clear the cached data from previous tests
`cmd+shift+R` then test it.

**Review by yeenab — APPROVED** [review]
### LGTM

**Pull request merged**